db/blob/blob_file_builder.cc

```diff
@@ -268,6 +272,13 @@ Status BlobFileBuilder::CloseBlobFile() {
                                      std::move(checksum_method),
                                      std::move(checksum_value));
 
+  assert(immutable_cf_options_);
+  ROCKS_LOG_INFO(immutable_cf_options_->info_log,
```

Comment:
Do you think it will be better and clearer to call out the fact in the LOG that the blob file is not synced at this point?

Reply:
It actually is synced by the time we get to this point (as a result of the `AppendFooter` call above).